diff --git a/selfdrive/car/honda/values.py b/selfdrive/car/honda/values.py
index a5779e9..94aa67f 100644
--- a/selfdrive/car/honda/values.py
+++ b/selfdrive/car/honda/values.py
@@ -28,8 +28,8 @@ class BP:
 
 AUDIO_HUD = {
   AudibleAlert.none: (BP.MUTE, CM.MUTE),
-  AudibleAlert.chimeEngage: (BP.SINGLE, CM.MUTE),
-  AudibleAlert.chimeDisengage: (BP.SINGLE, CM.MUTE),
+  AudibleAlert.chimeEngage: (BP.MUTE, CM.MUTE),
+  AudibleAlert.chimeDisengage: (BP.MUTE, CM.MUTE),
   AudibleAlert.chimeError: (BP.MUTE, CM.DOUBLE),
   AudibleAlert.chimePrompt: (BP.MUTE, CM.SINGLE),
   AudibleAlert.chimeWarning1: (BP.MUTE, CM.DOUBLE),
